src/UI/package.json

```diff
@@ -13,28 +13,32 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
+    "@apollo/client": "^3.7.0",
     "@auth0/nextjs-auth0": "^1.9.1",
     "@emotion/cache": "^11.10.1",
     "@emotion/react": "^11.10.0",
     "@emotion/server": "^11.10.0",
     "@emotion/styled": "^11.10.0",
+    "@hookform/resolvers": "^2.9.8",
+    "@material-ui/core": "^4.12.4",
     "@mui/icons-material": "^5.8.4",
     "@mui/material": "^5.9.3",
     "@reduxjs/toolkit": "^1.8.3",
     "@typescript-eslint/eslint-plugin": "^5.30.7",
     "axios": "^0.27.2",
-    "graphql": "^16.5.0",
-    "js-file-download": "^0.4.12",
```

Comment:
I think we want to be keeping this. At least for now as it's what we use to download data from the main page